models/user.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+const { hash } = require('./../helpers/hash');
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  User.init({
```

Comment:
is this table users does not have user id?

Reply:
The user_id is located in the migration file where I create a database table using Sequelize.